libraries/node-2fa/package.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Add here the author details from the original repo. We need to make sure we give credit to the author